app/channels/active_admin/chat/base_channel.rb

```diff
@@ -0,0 +1,28 @@
+module ActiveAdmin
+  module Chat
+    class BaseChannel < ApplicationCable::Channel
+      def subscribed
+        conversation ? stream_for(conversation) : reject
+      end
+
+      def speak(data)
+        message = conversation.public_send(
+          ActiveAdmin::Chat.message_relation_name.pluralize
```

Comment:
as you are already inside `ActiveAdmin::Chat` do you really need to specify `ActiveAdmin` module?

Reply:
I like to make it explicit to prevent Rails matching the incorrect class/module (it has happened to me 😅 )

Reply:
I'm fine with it then, just a down case I see for this is if want to re use Chat module outside Admin you would need to re write some implementation. But as this is an Admin Chat i don't see that need in the future